hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/keygen/TestCustomKeyGenerator.java

```diff
@@ -98,8 +105,9 @@ private TypedProperties getPropertiesForNonPartitionedKeyGen() {
   }
 
   @Test
-  public void testSimpleKeyGenerator() {
-    BuiltinKeyGenerator keyGenerator = new CustomKeyGenerator(getPropertiesForSimpleKeyGen());
+  public void testSimpleKeyGenerator() throws IOException {
```

Comment:
Can you run every tests twice. once with prop config and once with class config.

Reply:
> Can you run every tests twice. once with prop config and once with class config.

sure, will add it tomorrow

Reply:
> Can you run every tests twice. once with prop config and once with class config.

done, please take another look